src/main/io/displayport_msp_bf_compat.c

```diff
@@ -58,42 +58,44 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_DEGREES:
         return BF_SYM_GPS_DEGREE;
 
+/*
     case SYM_HEADING:
-        return BF_SYM_OVER_HOME;
+        return BF_SYM_HEADING;
+
+    case SYM_SCALE:
+        return BF_SYM_SCALE;
 
-/*    case SYM_SCALE:
-        return BF_SYM_SCALE; 
-*/
     case SYM_HDP_L:
-        return 'H';
```

Comment:
This one may be ok. Open for discussions.

HP has a meaning though, horse power, so does HD.

Reply:
Yeah, I agree that HP (or HD) aren't really the best abbreviations. But in the spirit of not changing the field's formatting to fit extra letters, IMO this is a suitable approximation. I know we won't get to the ideal case (which would be all the proper, customizable, icons), but the intention is to get as close as possible to that. Some compromises will be needed, though.